src/pages/use-cases/cluster-mesh.jsx

```diff
@@ -131,6 +132,24 @@ const testimonials = [
     quote: true,
     person: 'Luan Guimarães -  Site Reliability Engineer, Wildlife Studios',
   },
+  {
+    logo: SicrediLogo,
+    title: 'Strengthening Security Across Distributed Kubernetes Clusters',
+    CTAs: [
+      {
+        CTAtext: 'Watch The Talk',
+        url: 'https://youtu.be/MSqI-gBiCrc?si=ktIHXkxhuHpuwC6a',
+      },
+      // {
+      //   CTAtext: 'Read Case Study',
+      //   url: '',
+      // },
+    ],
+    description:
+      "Cilium Cluster Mesh gave us possibilities that we didn't have before. We can run the same application across our data centers and AWS. It provides us with a consistent networking experience wherever we need to go. Applications in different clusters can communicate with each other without needing to go through an ingress controller. We are expanding this capability for more applications because this value is already proven. Cluster Mesh works, it's fast, and it's reliable",
```

Comment:
really good quote choice

Reply:
I know right, it felt perfect for that page